samples/redis.js

```diff
@@ -0,0 +1,96 @@
+import { check } from "k6";
```

Comment:
Maybe let's add an `experimental` folder for samples that aren't for stable APIs :thinking: 


Also given that this requires a server I wonder if it shouldn't be in it's own folder and have a docker-compose.yml as well to run the redis server :thinking:

Reply:
Very fair point, will do just that 👍🏻